cloud_function/migrate_database.py

```
@@ -0,0 +1,586 @@
+        source_cursor.execute("""
+            SELECT sequence_name
+            FROM information_schema.sequences
+            WHERE sequence_schema = 'public';
+        """)
+
+        sequences = source_cursor.fetchall()
+
+        for (seq_name,) in sequences:
+            try:
+                # Get current value from source database
+                source_cursor.execute(f"SELECT last_value FROM \"{seq_name}\";")
```

Ruff (0.14.1)

Possible SQL injection vector through string-based query construction

(S608)